docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -167,6 +167,9 @@ The configuration parameters which must be set are:
   - This is set to the URI of the shared storage system to mount to the Alluxio root. This shared
   shared storage system must be accessible by all master nodes and all worker nodes.
   - Examples: `alluxio.underfs.address=hdfs://1.2.3.4:9000/alluxio/root/`, `alluxio.underfs.address=s3a://bucket/dir/`
+- `alluxio.master.journal.type=UFS`
+  - This uses UFS as the journal place. Note that Zookeeper cannot work 
+  with journal type `EMBEDDED` (use a journal embedded in the masters).
```

Comment:
But this here seems to suggest that embedded journal can't be used with zookeeper? I seem to be confusing myself